docs/python/1.6/python-appapi-devguide.md

```diff
@@ -46,8 +46,7 @@ Applications created with the IBM Streams Python API are executed on an instance
 
  - Use the **Streaming Analytics service running on IBM Cloud**: The Streaming Analytics service is a cloud version of IBM Streams, so you don't need to install Streams to build Python applications for the service. [Create a free instance of the Streaming Analytics service here](https://cloud.ibm.com/catalog/services/streaming-analytics).  The applications you create will run in the IBM Cloud.
 
-
- -  Enable the **IBM Streams add-on in IBM Cloud Pak for Data**: IBM Streams is included as an add-on in for IBM Cloud Pak for Data. Contact your administrator to enable the add-on.  Streams can also be installed as a stand-alone deployment on Red Hat OpenShift or Kubernetes environments.  
+ -  Enable the **IBM Streams add-on in IBM Cloud Pak for Data**: IBM Streams is included as an add-on in for IBM Cloud Pak for Data. Contact your administrator to enable the add-on.  Streams can also be installed as a stand-alone deployment on Red Hat OpenShift or Kubernetes environments.
```

Comment:
_IBM Streams is included as an add-on in for IBM Cloud Pak for Data._ Remove the "in" after add-on.